image-overview-templates/new-names/device-usb-camera-arm64.md

```diff
@@ -0,0 +1,26 @@
+$$./common/warning.md
+
+$$./common/quick-reference.md
+
+# Supported tags and respective Dockerfile links
+
+$$./common/version-kamakura.md |
+        - [Dockerfile](https://github.com/edgexfoundry/device-usb-camera/blob/v2.2.0/Dockerfile)
+
+# Quick reference
```

Comment:
```suggestion
# Quick reference (cont.)
```

Reply:
Humm, why do these have `(cont.)` ?? rhetorical... Go ahead and add for consistency and I'll clean up later once verify why.